framework/src/controller/bus.js

```diff
@@ -226,7 +226,7 @@ class Bus extends EventEmitter2 {
 		return Object.keys(this.events);
 	}
 
-	async cleanup() {
+	cleanup() {
```

Comment:
This function is missed to mention in the `BaseChannel` that we should add. And it should be kept as `async`, not just for this implementation rather for future refactoring and other implementations.